L1Trigger/L1TTrackMatch/python/L1TrackSelectionProducer_cfi.py

```diff
@@ -1,10 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
-L1TrackSelectionProducer = cms.EDProducer('L1TrackSelectionProducer',
-  l1TracksInputTag = cms.InputTag("L1GTTInputProducer","Level1TTTracksConverted"),
+l1tTrackSelectionProducer = cms.EDProducer('L1TrackSelectionProducer',
```

Comment:
Duplicate of
- cfipython/L1Trigger/L1TTrackMatch/l1TrackSelectionProducer_cfi.py

with the same search path

Reply:
Done